src/app/pages/progress/progress.page.ts

```diff
@@ -168,7 +169,7 @@ export class ProgressPage {
                     for (const k in task.responses) {
                       if (k === variableToGraph) {
                         // format the response time
-                        const response_time = moment(task.response_time).format(
+                        const response_time = moment(new Date(task.response_time)).format(
```

Comment:
Because, the task.response_time is in a depreciated format. it was causing a warning to be raised by the moment dependancy. the code changes nothing, other than just removed the warning by making sure the task.response_time is a proper and a not-depreciated date-time value.

Reply:
Can't moment_js work with timestamps directly? Why the conversion to  a JS date?

Reply:
It does work directly. But it gives this messages with it.

<img width="1090" alt="Screen Shot 2023-01-02 at 2 05 22 PM" src="https://user-images.githubusercontent.com/38203322/210234886-03629f22-c801-42ee-bde7-49b3849db924.png">

Reply:
It is better to convert the time to a JS date and then pass it to a moment for formatting. Otherwise, let us make sure the submitted date fields are in a moment-friendly format in the MomenTUM app and also in the json-maker, when they create the studies.